randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -400,61 +404,88 @@ Extra - room_id: [13]
   * Layers: default
   * L0 Hatch to Mimic Storage/Door to Frozen Tower
   * Extra - door_idx: (24,)
+  > Door to Data Room
```

Comment:
unsure whether we should add connections like this. i'm probably just worrying too much tho